prime-router/settings/organizations.yml

```diff
@@ -801,6 +801,44 @@
         port: 22
         filePath: ./upload
         credentialName: DEFAULT-SFTP
+    - name: elr-otc
+      organizationName: vt-doh
+      topic: covid-19
+      customerStatus: active
+      jurisdictionalFilter:
+        - "orEquals(ordering_facility_state, VT, patient_state, VT)"
+      qualityFilter:
+        - "AtLeastOneHasValue(Y, test_authorized_for_otc, test_authorized_for_home, test_authorized_for_unproctored)"
```

Comment:
This will not impact any current feeds initially. The defaults are applied based on a receiver setting. So we could turn that setting on for any receiver. I have always felt wary of framing the secondary feed as "the OTC feed". I don't feel like the default quality filters are specific enough and I really don't want to apply these defaults to any non-OTC result that happens to fail the filter. So I think we should make a separate receiver that only handles OTC where we can specify the type using that AtLeastOneHasValue filter looking at the otc fields. What do you think?